android/KMEA/build.sh

```diff
@@ -35,7 +35,9 @@ builder_describe "Builds Keyman Engine for Android." \
   "build" \
   "test             Runs lint and unit tests." \
   ":engine          Builds Engine" \
-  "--ci             Don't start the Gradle daemon. For CI"
+  "--ci             Don't start the Gradle daemon. For CI" \
+  --debug \
```

Comment:
Looking from the CI build, it seems like the dependency on `@/web` is only making a release build.  `--debug` doesn't get passed to `/web/build.sh`?

Reply:
True. `--debug` is an option that we want to be passing through to dependency builds. I need to do some work on that particular feature -- I'll make a sub-PR of this one.

I actually think we should never spec `--debug` in `builder_describe` but it should always be available, and _always_ be passed to child scripts and dependency scripts. (If the script doesn't use it, it doesn't hurt!)

Note: `-d` is shorthand for `--debug`.

`--debug` may be the only option we do that with for now, but there may be others in the future?